modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_well.yml

```diff
@@ -0,0 +1,39 @@
+id: az_paragraph_well
+label: AZ Paragraph Well
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+status: true
+
+source:
+  plugin: az_paragraphs_item
+  bundle: uaqs_well
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_cards
+
+process:
+
+  field_az_cards/body:
+    plugin: extract
+    source: field_uaqs_summary
+    index:
+      - 0
+      - value
+
+  field_az_cards/body_format:  
+    plugin: default_value
+    default_value: az_standard
+    
+  behavior_settings:
```

Comment:
This should be the same as the single card, correct?

Reply:
@trackleft thats correct!